GVRf/Framework/backend_oculus/src/main/java/org/gearvrf/GVRActivity.java

```diff
@@ -85,8 +91,14 @@ public final void setScript(GVRScript gvrScript, String dataFileName) {
 
     @Override
     protected void onInitAppSettings(VrAppSettings appSettings) {
-        if (mForceMonoscopic) {
-            appSettings.getMonoscopicModeParams().setMonoscopicMode(true);
+        if(mUseFallback){
+            if (android.os.Build.VERSION.SDK_INT < Build.VERSION_CODES.KITKAT) {
```

Comment:
Should this be "if (android.os.Build.VERSION.SDK_INT <**=** Build.VERSION_CODES.KITKAT" instead?